### PR TITLE
Add top bar for Join Firefox

### LINKED ISSUE #130: [Trailhead] The banner doesn't correspond to the specifications when using the Responsive Design Mode
[Notes]:
- This was tested on Desktop using the Responsive Design Mode since we have the website saved locally and cannot be tested on mobile.

[Affected versions]:
- Firefox Release 67 and up

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
- Have the static HTML/CSS/JS extracted to [.zip](https://drive.google.com/open?id=1i_Armc1yyZ8ogxrPqTXzMcGxzduOwJoM).

[Steps to reproduce]:
1. Open the browser.
2. Open the index.html file with the browser.
3. Press Ctrl+Shift+M keys.
4. Observe the way the text is displayed inside the banner.

[Expected result]:
- The "Join Firefox. Learn more" strings are displayed on the left side, on the second row.

[Actual result]:
- The "Join Firefox. Learn more" strings are displayed on the first row.

[Additional Notes]:
- Here are the [mock-ups](https://mozilla.invisionapp.com/share/ASS7BZ9GUYP#/screens/365354398) describing the banner in mobile view.
- Attached a screenshot of the issue:

Lockwise mobile

### PR DESCRIPTION
**Pull request by devinreams**
image

Screen Shot 2019-05-31 at 07 58 55

- [x] add banner to home page (only) with tracking link
- [x] fix bug with flex re-calculating the Firefox logo instead of keeping it 24px on small responsive layout
- [x] change from a fixed position over the page to actually block content pushing the page content down
- [x] make sure "Join Firefox" wraps together onto new line Fixes #130

---

tldr on how to test locally, see readme for more details:

- clone this repo
- check out this branch
- run the `bundle exec jekyll serve` commands

### CODE REVIEW

**Comment by changecourse** [comment]
If possible, I’d prefer for the top bar to sit above the main content, rather than over it, based on spacing overlaps in the screenshot posted above.

**Comment by devinreams** [comment]
@changecourse OK. I quickly tried that but broke all kinds of layout based on how the page is built with flex. Will play a bit more with it but that was the quickest implementation to get to staged.

**Comment by devinreams** [comment]
Tagging @johngruen in case he can help out since this is his website 🏅 and much smarter at CSS than I am. Thanks for the extra eyes 🎩

**Comment by devinreams** [comment]
Here's the latest based on @johngruen's speedy work ⚡️ 


image

Screen Shot 2019-05-31 at 07 58 55
